cli/tests/integration/check_tests.rs

```diff
@@ -189,18 +202,21 @@ fn typecheck_core() {
     ),
   )
   .unwrap();
-  let output = util::deno_cmd_with_deno_dir(&deno_dir)
-    .arg("run")
-    .arg(test_file.to_str().unwrap())
-    .output()
-    .unwrap();
-  println!("stdout: {}", String::from_utf8(output.stdout).unwrap());
-  println!("stderr: {}", String::from_utf8(output.stderr).unwrap());
-  assert!(output.status.success());
+
+  let output = context
+    .new_command()
+    .args(format!("run {}", test_file.to_str().unwrap()))
```

Comment:
Unfortunately for now, we'll need to go through and change all of these that provide an absolute file path to use `.args_vec` because this won't properly escape when there's a space in the directory name. In the future we can develop something better (maybe have a macro that properly escapes strings to arguments when provided like this)

Reply:
For the coverage tests it's a bit tricky to use `args_vec` because then a space is added after `--coverage=`, breaking the command. I can fix it with 
```rs
   .args(vec![
      "test ",
      "--quiet ",
      "--coverage=", 
      tempdir.to_str().unwrap(),
    ].join(""))
 ```
 But it doesn't seem very elegant. Is this OK or is there a better way?

Reply:
We need to do `.args_vec` because `.args` will will break if `tempdir.to_str()` has a space in it. It should be like this:

```
   .args_vec(vec![
      "test".to_string(),
      "--quiet".to_string(),
      format!("--coverage={}", tempdir.to_str().unwrap()),
    ])
```

We can work on making that less verbose in the future.

Reply:
We could also add a `.arg(...)` method to `TestCommandBuilder` so the code is similar to before.